pkg/schedule/checker/rule_checker.go

```diff
@@ -384,7 +384,7 @@ func (c *RuleChecker) allowLeader(fit *placement.RegionFit, peer *metapb.Peer) b
 	if s == nil {
 		return false
 	}
-	stateFilter := &filter.StoreStateFilter{ActionScope: "rule-checker", TransferLeader: true}
+	stateFilter := &filter.StoreStateFilter{ActionScope: "rule-checker", TransferLeader: true, OperatorLevel: constant.High}
```

Comment:
If we have set fastFailover, the operator level might be different.

Reply:
the allow leader doesn't check store limit, so the priority can be ignore.